apm-collector/apm-collector-boot/src/main/resources/application.yml

```diff
@@ -86,4 +86,11 @@ configuration:
     applicationAverageResponseTimeThreshold: 2000
 # thermodynamic
     thermodynamicResponseTimeStep: 50
-    thermodynamicCountOfResponseTimeSteps: 40
\ No newline at end of file
+    thermodynamicCountOfResponseTimeSteps: 40
+# TODO: receiver_zipkin need to remove before merge, and only provide this in document.
+receiver_zipkin:
```

Comment:
Is it necessary to pass this information being structured? Can't you just pass the `endpoint` (e.g. `endpoint: http://localhost:9411/api/v2/spans`  as we usually do for zipkin? cc @adriancole @wu-sheng

Reply:
Which parts do you refer? `receiver_zipkin`?

Reply:
@jcchavezs I guess you mean, ip, port, context path, servlet mapping path, etc. That is because I don't use an URL util to analysis the endpoint string. Does Zipkin do this by providing an URL endpoint only? If this is a tradition, I can follow that.